git-hooks/README.md

```diff
@@ -0,0 +1,20 @@
+# Git Hooks
+
+This directory contains hooks to avoid pushing broken or unpolished code to CI/CD, which can cause some noisy notifications.
+
+## Configuration
+
+To use them, you'll need to set your [gitconfig](https://git-scm.com/docs/githooks#_description) to include this directory for hooks. To do that for this repo only, run:
+
+```
+echo '[core]' >> .git/config;
+echo '  hooksPath="./git-hooks"' >> .git/config;
```

Comment:
Since this requires a change to dev configuration, we should run this by @ilodidoj.

Perhaps discuss in the next pairing?